build/ci_utils/src/actions/artifacts/raw.rs

```diff
@@ -139,7 +139,8 @@ pub mod endpoints {
         let is_gzipped = response
             .headers()
             .get(reqwest::header::ACCEPT_ENCODING)
-            .contains(&HeaderValue::from_static("gzip"));
+            .as_ref()
+            .is_some_and(|h| h == &HeaderValue::from_static("gzip"));
```

Comment:
This is so much worse to read than previous version. Is `contains` removed? What lint forces this change?

Reply:
See https://github.com/enso-org/enso/pull/7109#discussion_r1238439069 you may look at contains feature discussion in rust.

Reply:
I did implement the replacement in enso-prelude, but it is not available in build scripts. Personally I don't see it as bad.